changelogs/fragments/1684-Add_validation_for_marker_begin_end.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - zos_blockinfile - User could set same values for marker begin and end generate issue on ZOAU python API.
+    Change now validates that ``marker_begin`` and ``marker_end`` are different and fails if they are the same.
+    (https://github.com/ansible-collections/ibm_zos_core/pull/1684).
```

Comment:
An Ansible user won't care particularly about bugs in ZOAU (or any other dependencies). Better to use this space to describe the bug as experienced by the Ansible user. 

> User could previously set the same values for module options 'marker_begin' and 'marker_end', which would result in <..whatever the issue was..>. Fix now introduces a requirement for 'marker_begin' and 'marker_end' to have different values.

Even better would be if you could bring the <..whatever the issue was..> to the beginning, 
> Previously <..issue..> happened when 'marker_begin' and 'marker_end' were set to the same value. Fix introduces a requirement for 'marker_begin' and 'marker_end' to have different values.